submarine-workbench/workbench-web-ng/src/app/services/base-api.service.ts

```diff
@@ -35,7 +35,8 @@ class HttpError extends Error {
     this.params = params;
 
     if (!environment.production) {
-      this.logError();
+      // comment out because weird this behavior
+      // console.log(this);
```

Comment:
```suggestion
      this.logError();
```

Reply:
This code will produce unexpected behavior. Refer to [this issue](https://github.com/Microsoft/TypeScript-wiki/blob/master/Breaking-Changes.md#extending-built-ins-like-error-array-and-map-may-no-longer-work).